aodndata/moorings/products_handler.py

```diff
@@ -132,6 +159,14 @@ def _make_aggregated_timeseries(self):
             product_file.publish_type = PipelineFilePublishType.HARVEST_UPLOAD
             self.file_collection.add(product_file)
 
```

Comment:
I would not expect that a "make" method would delete files. I think it's better to be explicit in the call to remove - maybe a cleanup method or renaming here.

Reply:
Ok, I've factored out the "delete" code in a new method, but it's still easiest to call it here, where the new product just got created, as I have all the information I need.
Note that the actual publishing and deleting happens in a later `publish` step, based on the attributes of each file in the `file_collection` (see https://aodn.github.io/python-aodncore/userdoc.html)